lib/client.c

```diff
@@ -1054,14 +1058,17 @@ SF_STATUS STDCALL snowflake_set_attribute(
             break;
         case SF_CON_LOGIN_TIMEOUT:
             sf->login_timeout = value ? *((int64 *) value) : SF_LOGIN_TIMEOUT;
-            if (sf->login_timeout < SF_LOGIN_TIMEOUT)
-            {
-              sf->login_timeout = SF_LOGIN_TIMEOUT;
-            }
             break;
         case SF_CON_NETWORK_TIMEOUT:
-            sf->network_timeout = value ? *((int64 *) value) : SF_NETWORK_TIMEOUT;
+            sf->network_timeout = value ? *((int64 *) value) : SF_LOGIN_TIMEOUT;
```

Comment:
again why we are changing other timeout?

Reply:
roll back the change made in previous PR to keep the behavior of existing connection parameters.